### DISCUSSION TITLE
Action after Stop or Pause (pre-compiled vanilla E3v2)

### DISCUSSION THREAD

**Discussion by TheRiflesSpiral** [Jyers/Marlin#456] [Q&A] [answered]
In stock Creality firmware, a Pause or Stop of a print raises z about 10mm and moves X/Y to 0,0.

The pre-compiled Jyers/Marlin firmware appears to simply stop.

I print with OctoPrint but Stop/Pause from the printer.

Is there any way to get the previous behavior back?

Great firmware, by the way! The cooperation between OctoPrint and the Printer Display is exactly what I was hoping for.

**Comment by Jyers** [comment, marked as answer]
I left this alone to give the user control over what the printer does on a cancel. In octoprint you can define an "after cancel" script in the gcode scripts section. Here you can tell the printer to move up and away.

**Reply by TheRiflesSpiral** [reply]
Thanks for the feedback. I'll update that script.

Will that GCode run when stopping/pausing from the display?

**Comment by aryajuanda** [comment]
its a octoprint behavior what to do on stop/cancel... on stock creality you cant stop/cancel from LCD display...

just set on the octoprint gcode what todo on pause/cancel

**Reply by TheRiflesSpiral** [reply]
**on stock creality you cant stop/cancel from LCD display...**

Of course you can. I've paused/stopped prints dozens of times from the display before changing the firmware. The Tune/Pause/Stop menu on the stock Creality firmware is exactly the same as Jyers.

**Reply by Jyers** [reply]
I believe @aryajuanda means you can't stop/cancel and octoprint job on the stock firmware. I added that functionality along with the octoprint support.

**Reply by aryajuanda** [reply]
yup... only SD print can be pause/stop from LCD with official Creality firmware.. not an octoprint job.. 
i m stating using this firmware because of this feature.. and i love it so much..

**Reply by TheRiflesSpiral** [reply]
Oh, I see. I only every tried once to print via Octoprint with the stock Firmware. When I saw that the display wasn't responsive I just decided to use an SD card workflow.
